server/base/src/test/java/org/apache/accumulo/server/client/BulkImporterTest.java

```diff
@@ -147,25 +148,25 @@ public void testFindOverlappingTablets() throws Exception {
       List<TabletLocation> overlaps =
           BulkImporter.findOverlappingTablets(context, vm, locator, new Path(file), null, null, cs);
       assertEquals(5, overlaps.size());
-      Collections.sort(overlaps);
-      assertEquals(new KeyExtent(tableId, new Text("a"), null), overlaps.get(0).tablet_extent);
+      Collections.sort(overlaps, Comparator.comparing(tl -> tl.getExtent()));
```

Comment:
The previous Comparable compared by extent, then by location, then by session. A few questions:

1. Did those secondary sorts not really matter?
2. Did I miss something, or is this test the only time we care about these being ordered? Was that Comparable stuff only there for this test?

Reply:
Interested in this as well. Other than this, LGTM.

Reply:
Yeah this test was the only thing that used the compare to functionality.  So I thought it would be simpler from a maintenance stand point to move it to the test.  Also the test only needs to order by extent, it does not need any secondary comparison.